config/jobs/kubeflow/kubeflow-postsubmits.yaml

```diff
@@ -235,6 +235,8 @@ postsubmits:
         - "7200"
     annotations:
       testgrid-dashboards: sig-big-data
+      testgrid-alert-email: cloud-ml-pipeline-oncall@google.com
```

Comment:
This email doesn't exist yet, right? (If we were to setup a new one, maybe name it with kubeflow instead of cloud)

I wonder if there's any concern for exposing an email to the public. Is is possible to monitor this from inside Google?

/cc @Bobgy

Reply:
I replaced the email by kubeflow-engineering@google.com.  Please let me know your thought :-)